app/javascript/packages/address-search/components/full-address-search-input.tsx

```diff
@@ -82,12 +83,20 @@ export default function FullAddressSearchInput({
     [addressValue, cityValue, stateValue, zipCodeValue],
   );
 
+  const getErroneousAddressChars = () => {
+    const errChars = addressValue.replace(/[A-Za-z0-9\-' .#]*/g, '');
```

Comment:
Similar point as previous, maybe we could assign the regular expression to a constant and reference it in both places in this file.